Automattic/wp-calypso
# Update Reader Post Component in SEO Reader Preview

Previously, we were building our own reader post layout by using the same css classnames as the Reader. Now that we have the `reader-post-card` component, we can hook into that for a less painful way of recreating a Reader post in the SEO preview.

We still need to do some work to convert the editor's `post` object into a `readerpost` object by adding some properties to it.

**To test**
* Ensure you have a business plan
* Visit the editor for a multitude of posts (text only, with images, image only, etc)
* Click the preview button, and verify that the reader preview in the SEO tab looks as expected.
* Note: None of the links/buttons should be clickable in the Reader preview.

Screenshots:
<img width="865" alt="screen shot 2017-03-31 at 11 46 53 am" src="https://cloud.githubusercontent.com/assets/789137/24565128/be18b22e-1608-11e7-8b76-11ccb67f32bd.png">
<img width="862" alt="screen shot 2017-03-31 at 11 33 38 am" src="https://cloud.githubusercontent.com/assets/789137/24565126/be1735ca-1608-11e7-8946-725f6ac2e2b9.png">
<img width="863" alt="screen shot 2017-03-31 at 11 43 00 am" src="https://cloud.githubusercontent.com/assets/789137/24565127/be17c274-1608-11e7-85d9-c7d9f89a5379.png">


Fixes #12024.
